modules/nf-core/manta/tumoronly/main.nf

```diff
@@ -33,8 +33,9 @@ process MANTA_TUMORONLY {
     configManta.py \
         --tumorBam $input \
         --reference $fasta \
+        --runDir manta \
         $options_manta \
-        --runDir manta
+        $args
 
     python manta/runWorkflow.py -m local -j $task.cpus
```

Comment:
then here too?

Reply:
no I don't think we can pass args to this one

Reply:
ah never mind

Reply:
sorry need more coffee